.github/workflows/codespell.yml

```diff
@@ -11,5 +11,5 @@ jobs:
       - uses: actions/checkout@v2
       - uses: codespell-project/actions-codespell@master
         with:
-          skip: ./bin/trace,./hl/tools/h5watch/h5watch.c,./tools/test/h5jam/tellub.c
+          skip: ./bin/trace,./hl/tools/h5watch/h5watch.c,./tools/test/h5jam/tellub.c,./config/sanitizer/LICENSE
```

Comment:
codespell thinks "complies" should actually be "compiles" and I'm disinclined to change the Apache LICENSE

Reply:
it doesn't think "complies" is a word?

Reply:
Or is it trying to infer which it should be from grammar rules or something?